tools/topology/topology2/platform/intel/hdmi-in-ssp-config.conf

```diff
@@ -0,0 +1,54 @@
+Object.Dai.SSP [
```

Comment:
Not following how SSP and HDMI are related here, it looks like SSP is consuming HDMI clock if so we should spell it out in a comment.

Reply:
@lgirdwood : The use case is HDMI input capture via I2S pins. .i.e. the converter chip will receive HDMI input and send the audio pcm  samples via I2S pins to Intel SOC. In this case I2S is running on slave mode and MCLK supplied  by converter chip.

Reply:
@lgirdwood : Added the comment in file to under stand the use case and configuration.

Reply:
Thanks @CBala21 got you - @plbossart is the I2S expert so best for him to review too.